railties/Rakefile

```diff
@@ -94,6 +96,8 @@ namespace :test do
       test_files = buckets[n]
     end
 
+    OUTPUT_FILE = Tempfile.new("railties_test_reporter")
```

Comment:
We shouldn't define constants in Rake tasks like this, they end up in `Object::`.

Reply:
It doesn't have to be, I was just trying to reference it inside the Minitest plugin but it didn't work hence setting the `ENV` var.

Open to better suggestions there, but this was the cleanest way I could come up with.

Reply:
I think you can just use a hardocded path in `/tmp`. This only runs on CI so it's a controlled environment.